package/MDAnalysis/coordinates/chemfiles.py

```diff
@@ -333,7 +347,12 @@ def _timestep_to_chemfiles(self, ts):
         if ts.has_velocities:
             frame.add_velocities()
             frame.velocities[:] = ts.velocities[:]
-        frame.cell = chemfiles.UnitCell(*ts.dimensions)
+
```

Comment:
If this is the only real change, could we have a try/except here and allow 0.9 and 0.10?

Reply:
This is the only change in the part of the API used in MDA yes.

I could add a version check here, but for the future I'd like to not keep old versions around too long, and drop 0.9 when 0.11 comes out 😃

Reply:
Either way this goes, can the minimum version be pinned in the gh actions CI workflow? That way we can be sure we're picking up the right thing.